src/hockeypuck/ratelimit/cmd/ratelimit-tester/test-key.asc

```diff
@@ -0,0 +1,29 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Is this meant to be a genuine key or nonsense? It seems to be a mishmash of genuine packets and hex gibberish ("Aa1Bb2Cc3..."). Can we use one of the existing test keys instead?